L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -151,9 +151,9 @@
 from L1Trigger.L1TTrackMatch.l1tTrackerEtMiss_cfi import *
 from L1Trigger.L1TTrackMatch.l1tTrackerHTMiss_cfi import *
 # make the input tags consistent with the choice L1VertexFinder above
-l1tTrackJets.L1PVertexCollection  = ("l1tVertexFinder", "l1vertices")
+l1tTrackJets.VertexInputTag  = ("l1tVertexFinderEmulator","l1verticesEmulation")
 l1tTrackFastJets.L1PrimaryVertexTag  = ("l1tVertexFinder", "l1vertices")
-l1tTrackJetsExtended.L1PVertexCollection  = ("l1tVertexFinder", "l1vertices")
+l1tTrackJetsExtended.VertexInputTag  = ("l1tVertexFinderEmulator","l1verticesEmulation")
```

Comment:
Should these be `InputTag`s as well?

Reply:
`InputTag`s can be assigned to from a tuple of strings. In general we recommend to not specify the types explicitly on the right hand side, because then the framework ensures the assigned-to configuration parameter (`VertexInputTag` in this case) already exists, and has a compatible type.

Reply:
Dear all, 

So I dont change anything here right? Or should I change the VertexInputTag?